tests/integration/runners/test_state.py

```diff
@@ -93,6 +93,15 @@ def test_orchestrate_nested(self):
         assert os.path.exists('/tmp/ewu-2016-12-13') is False
         assert code != 0
 
+    def test_orchestrate_with_mine(self):
+        '''
+        test salt-run state.orchestrate with mine.get call in sls
+        '''
+        test = self.run_run('mine.update "*"')
```

Comment:
Mine updates are async. This might race.

Reply:
okay looking into a solution to make sure this doesn't occur.